env-check.sh

```diff
@@ -89,7 +89,7 @@ case "$SUDO_LEVEL" in
         if [[ "$SUDO_LEVEL" -eq 2 ]]; then
             [[ ! "$SUDO_PASSWD" ]] &&  check_res=1 && out_str=$out_str"\n
 \tPlease setup \e[31mSUDO_PASSWD\e[0min $(dirname $0)/case-lib/config.sh file\n
-\t\tIf you don't want modify this value, you will need to export SUDO_PASSWD\n
+\t\tIf you don't want modify to this value, you will need to export SUDO_PASSWD\n
```

Comment:
'want to modify'

Reply:
Actually, this is correct --- it is saying if you dont want to  save the password.